project-2-voting/anchor/programs/voting/src/lib.rs

```diff
@@ -98,10 +108,11 @@ pub struct Vote<'info> {
     pub poll_account: Account<'info, PollAccount>,
 
     #[account(
-        mut,
+        mut, // Vote() will modify a given candidate
```

Comment:
I think this should be emphasized. The original code in [video](https://youtu.be/amAq-WHAFs8?list=PLilwLeBwGuK7HN8ZnXpGAD9q6i4syhnVc&t=6513) also made such a mistake.